admin_guide/cluster_capacity.adoc

```diff
@@ -143,6 +143,12 @@ $ oc adm policy add-cluster-role-to-user cluster-capacity-role \
     system:serviceaccount:default:cluster-capacity-sa
 ----
 
+[NOTE]
+====
+The `default` specification in the above command refers to the `default`
+project. If using an alternative project, amend the command accordingly.
```

Comment:
s/using/you use

> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”